.env.dist

```diff
@@ -0,0 +1,43 @@
+# Zend settings
+# Use production or development
+APP_ENV=production
+
+# Nginx settings
+# Use off or "\"Restricted\"" (or another escaped string); also required an entry in docker/nginx/.htpasswd
+NGINX_REQUIRE_AUTH=off
+
+# Mail settings for production
+DOCKER_MAIL_HOST=smtp.tue.nl
```

Comment:
![image](https://user-images.githubusercontent.com/17340320/114285018-34e0a500-9a54-11eb-92d8-d5cffa46e3e9.png)

Please note that `smtp.tue.nl` no longer is available without authentication from non-whitelisted hosts such as `gewis.win.tue.nl`, `gewisweb.win.tue.nl`  and `gewiscontainer.win.tue.nl`. The CBC created a workaround for this on `gewis.win.tue.nl` by routing it through the whitelisted host `gewismail.win.tue.nl` but for security reasons they do not allow unauthenticated mail delivery to non-GEWIS domains from hosts that aren't deprecated. 

For applications that send large volumes of mail, the CBC suggests to include a private Postfix TLS relay (such as [juanluisbaptiste/postfix](https://hub.docker.com/r/juanluisbaptiste/postfix)). An example config can be found at the bottom of this comment. This relay can then forward the mail to the authenticated mail server (optionally also updating To/From headers)

If you do not send large volumes of mail, or use an internal queue (you can reuse the connection for efficiency), you can also implement direct connections to `mail.gewis.nl:587` in your application to keep the setup simple. This also allows you to get feedback about the mail delivery in your application which you otherwise would not have. 

```yaml
postfix:
  hostname: web-postfix
  image: juanluisbaptiste/postfix
  environment:
    - SMTP_SERVER=smtp.gewis.nl
    - SMTP_USERNAME=service-web@gewis.nl
    - SMTP_PASSWORD=TODOTODO
    - SMTP_PORT=587
    - SERVER_HOSTNAME=web-postfix
  restart: always
  stop_grace_period: 90s            # We want to make sure all emails are sent before the data is potentially destroyed
  networks:
    web-network:
      aliases:
        - web-postfix
  expose:
    - "25"
```

Reply:
Zend could not handle the TLS connection up to my expectations, so I went with the postfix setup after all. Thanks for the advice. I now tested it, and it works.